backends/deeploy/pact_export.py

```diff
@@ -140,7 +151,7 @@ def export_net(net: nn.Module,
                name: str,
                out_dir: str,
                eps_in: float,
-               in_data: torch.Tensor,
+               in_data: Tuple[torch.Tensor],
```

Comment:
Please check with other tests whether this is not breaking - I'd prefer Union[torch.Tensor, Tuple[torch.Tensor, ...]]!

Reply:
Done in [this commit](https://github.com/pulp-platform/quantlib/pull/15/commits/c2553bef1a29dc0baf6dd19cf1cbd8a2a351cedf)